### PR TITLE
[release-1.2] Bump VM_IMPORT to v0.2.4

### PR DESCRIPTION
**Pull request by tiraboschi**
Bump VM_IMPORT to v0.2.4

Signed-off-by: Simone Tiraboschi <stirabos@redhat.com>

**Release note**:
```release-note
Bump VM_IMPORT to v0.2.4
```

### CODE REVIEW

**Comment by ovirt-infra** [comment]
All tests passed

File: deploy/olm-catalog/kubevirt-hyperconverged/1.2.0/kubevirt-hyperconverged-operator.v1.2.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -2295,9 +2295,9 @@ spec:
     name: virt-launcher
   - image: docker.io/kubevirt/virt-operator@sha256:fec75f1d53426826a6e7b3e78f89730970dc3f7cfac7f44de7079247e4891492
     name: virt-operator
-  - image: quay.io/kubevirt/vm-import-controller@sha256:59c3f2a65fbac003a772c1850a1d819b0ee529763bf2e199c2d3a799c44e71e4
+  - image: quay.io/kubevirt/vm-import-controller@sha256:8ee55747671cd78b02a05a4b6a18737c8241c6ad7e6456d1d4a92e52f6c10dad
     name: vm-import-controller
-  - image: quay.io/kubevirt/vm-import-operator@sha256:62706b1ce8df1d4427f591f0b6a75a73560e724d8d5fc860144467ecf5be1edc
+  - image: quay.io/kubevirt/vm-import-operator@sha256:1f90a44eb9a567c2e3ca6b50ca8643d850a2d04b7f9a82992ad7e6777f84ec30
```
Comment (nunnatsa) [inline deploy/olm-catalog/kubevirt-hyperconverged/1.2.0/kubevirt-hyperconverged-operator.v1.2.0.clusterserviceversion.yaml:2300]: missing `vm-import-virtv2v`

**Review by nunnatsa — CHANGES_REQUESTED** [review]
missing `vm-import-virtv2v`

File: deploy/olm-catalog/kubevirt-hyperconverged/1.2.0/kubevirt-hyperconverged-operator.v1.2.0.clusterserviceversion.yaml (hunk 1 above)
Comment (tiraboschi) [reply deploy/olm-catalog/kubevirt-hyperconverged/1.2.0/kubevirt-hyperconverged-operator.v1.2.0.clusterserviceversion.yaml:2301]: done, thanks

**Comment by ovirt-infra** [comment]
All tests passed

**Review by nunnatsa — APPROVED** [review]
/approve

**Pull request merged**